saleor/dashboard/forms.py

```diff
@@ -33,24 +43,58 @@ def valid_value(self, value):
         forms.Field.validate(self, value)
         return True
 
-    def set_initial(self, obj):
+    def set_initial(self, obj, obj_id=None, label=None):
         """Set initially selected objects on field's widget."""
-        selected = {'id': obj.pk, 'text': str(obj)}
+        selected = {
+            'id': obj_id if obj else obj.pk,
+            'text': label if label else str(obj)}
         self.widget.attrs['data-initial'] = json.dumps(selected)
 
 
+class AjaxSelect2CombinedChoiceField(AjaxSelect2ChoiceField):
+    """An AJAX-based choice field using Select2 for multiple querysets.
+
+    <obj.pk>_<obj.__class__.__name__> represents value passed to a field.
```

Comment:
I'm not sure how to read this. What is `obj` here? Maybe an example would make it easier to parse?